### PR TITLE
Feature/hashed password

### PR DESCRIPTION
**Pull request by nietrenswens**

### CODE REVIEW

File: client/src/pages/LoginScreen/LoginScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        setError(error.response.data);
+      } else {
+        setError("Er is iets misgegaan tijdens het inloggen.");
+      }
+    }
+  }
+
+  // If already logged in, redirect to dashboard
+  if (isAuthenticated) {
+    console.log("je moeder");
```
Comment (Failfan13) [inline client/src/pages/LoginScreen/LoginScreen.tsx:104]: Nee die van jouwwwww *insert cakedup pikachu holding uno reverse card*

**Review by Failfan13 — CHANGES_REQUESTED** [review]
I like yes yes many many of me own written code verry cewl

File: client/src/pages/LoginScreen/LoginScreen.tsx (hunk 1 above)
Comment (nietrenswens) [reply client/src/pages/LoginScreen/LoginScreen.tsx:104]: Ehhh die is niet van mij...

**Comment by nietrenswens** [comment]
This PR is to the wrong branch. Closing

**Pull request closed without merging**